.github/workflows/static_analysis.yml

```diff
@@ -1,6 +1,8 @@
 name: Static Code Analysis
 
-on: [pull_request]
+on:
+  pull_request:
+    types: [opened, reopened, synchronize]
```

Comment:
Is this different from the defaults? According to https://frontside.com/blog/2020-05-26-github-actions-pull_request/#how-do-workflows-trigger-on-pull_request this isn't different. It may be nicer to be explicit than implicit though.

Reply:
oh oops. Didn't realize that. I'll just keep it the way it is for now but now I know for the future ^_^